analyzer/src/main/resources/init.gradle

```diff
@@ -32,7 +32,11 @@ import org.gradle.tooling.provider.model.ToolingModelBuilderRegistry
 // See:
 // https://docs.gradle.org/current/userguide/init_scripts.html#sec:init_script_plugins
 // https://github.com/gradle/gradle/blob/176c18d/subprojects/docs/src/samples/toolingApi/customModel/plugin/src/main/java/org/gradle/sample/plugin/CustomPlugin.java
-apply plugin: DependencyTreePlugin
+if (AbstractDependencyTreePlugin.isCompatibleGradleVersion(gradle.gradleVersion)) {
+    apply plugin: DependencyTreePlugin
+} else {
+    apply plugin: LegacyDependencyTreePlugin
```

Comment:
I guess it is not desirable to abort the process here. It's confusing why the two classes `DependencyTreePlugin` and `LegacyDependencyTreePlugin` are necessary since they don't seem to implement any behaviour. I know nothing about how Gradle plugins work though. :)

Reply:
The reason is given in the commit message, but I will add a code comment in a separate PR. It's because before 2.14 it had to be a project scope plugin, and starting with 2.14 a gradle scope plugin.